[Dev Environment] `pnpm dev` fails with `ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL` on root

## Bug report
When running `pnpm dev` to start Storybook for contributing, the command fails with `ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL`.

### Current Behavior

```
<PRIMITIVES_ROOT>/node_modules/.pnpm/storybook@9.1.15_@testing-library+dom@10.4.1_prettier@3.6.2_vite@7.1.12_@types+node@22.14.0_terser@5.39.0_/node_modules/storybook/bin/index.cjs:23
  throw error;
  ^

Error: spawn none ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:285:19)
    at onErrorNT (node:internal/child_process:483:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)
Emitted 'error' event on ChildProcess instance at:
    at ChildProcess._handle.onexit (node:internal/child_process:291:12)
    at onErrorNT (node:internal/child_process:483:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn none',
  path: 'none',
  spawnargs: [ 'http://localhost:6006/' ]
}

Node.js v22.18.0
<PRIMITIVES_ROOT>/apps/storybook:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @repo/storybook@0.0.0 dev: `storybook dev -p 6006 -- -p 9009`
Exit status 7
```

### Expected behavior

Storybook should start successfully.

### Suggested solution
This issue appears to be a bug in Storybook, which might be fixed in `v10.3.0-alpha.3`. However, since that version is still in alpha, I suggest a workaround.
ref. https://github.com/storybookjs/storybook/pull/33730

We can use `--no-open` option for `storybook dev` instead of `BROWSER=none` at [package.json](https://github.com/radix-ui/primitives/blob/907513701a75b11a115563f9554ac6e8147bf2db/package.json#L11).
ref. https://storybook.js.org/docs/api/cli-options#dev

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Node             | n/a     | 22.18.0 |
| npm/yarn/pnpm    | pnpm | 9.15.9 |
| Operating System | Ubuntu (WSL2) | 24.04.1 LTS |
| Shell | Bash | 5.3.9 |

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "types:check": "pnpm -r --parallel run typecheck",
     "test": "vitest",
     "test:ci": "vitest run && pnpm cypress:ci",
-    "storybook": "BROWSER=none pnpm --filter=@repo/storybook run dev -p 9009",
+    "storybook": "pnpm --filter=@repo/storybook run dev -p 9009 --no-open",
     "storybook:build": "BROWSER=none pnpm --filter=@repo/storybook run build",
     "storybook:serve": "pnpm --filter=@repo/storybook run serve",
     "cypress:ci": "start-server-and-test 'pnpm dev' http://localhost:9009 'pnpm cypress:install && pnpm cypress:run'",

